src/Service/NotificationServiceInterface.php

```diff
@@ -19,5 +19,8 @@ interface NotificationServiceInterface
     /**
      * @throws NotificationException
      */
-    public function send(NotificationInterface $notification): int;
+    public function send(
+        NotificationInterface $notification,
+        string $groupCode = NotificationService::GROUP_CODE_DEFAULT
```

Comment:
The constant `GROUP_CODE_DEFAULT` must be set in the interface, not in the concrete `NotificationService` implementation.